cluster-provision/k8s/1.19/provision.sh

```diff
@@ -316,6 +316,24 @@ echo "tmpfs /var/provision/kubevirt.io/tests tmpfs rw,context=system_u:object_r:
 
 dnf install -y NetworkManager-config-server
 
+# Add CNI DHCP daemon service, disabled by default
```

Comment:
Why is it necessary to create this service, cant we just enable it?

Reply:
I'm not sure I understand your question.
This is creating a systemd service for `/opt/cni/bin/dhcp daemon`, since I could not find any existing one.
Since it is just in `/etc/systemd/system`, it gets added to the list of systemd available services but it is not enabled by default.
The idea is to enable it only when we need it, i.e. when creating CNAO lanes that potentially have multiple NICs.
Without that daemon, CNI can't do DHCP on pods using secondary interfaces.

Reply:
Thanks for explaining I understand this better now,
I meant if there is already an existing service on systemd cant we just enable and start it when needed? (e.g CNAO tests suite) what did I miss?

Reply:
There is no such service, it was discussed in the other thread.